index.js

```diff
@@ -40,11 +40,42 @@ function defaultFormatter(messages) {
   return output.trim();
 }
 
+// load custom rles
+function loadCustomRules(rulesdir) {
```

Comment:
Could this function be replaced with a call to [node-glob](https://github.com/isaacs/node-glob)?